### ISSUE TITLE
Truvari collapse not producing any collapsed variants

### ISSUE THREAD

**Issue by vondant** [ACEnglish/truvari#233]
Merging the variants of the same sample from 2 different variant callers, namely DELLY and MANTA, is generating no collapsed variants.

I ran 
`bcftools merge -m none --force-samples delly.vcf.gz manta.vcf.gz | bgzip merged.vcf.gz`

then

```
    "input": "/newhome/GDC-Pipeline-Output/TCGA_WGS_Swag_Analysis/TCGA_Validation_2/TCGA-EJ-8474/1/somatic-sv-merging_v3.0/TCGA-EJ-8474_merged.vcf.gz",
    "output": "/newhome/GDC-Pipeline-Output/TCGA_WGS_Swag_Analysis/TCGA_Validation_2/TCGA-EJ-8474/1/somatic-sv-merging_v3.0/TCGA-EJ-8474_truvari_merged.vcf",
    "collapsed_output": "/newhome/GDC-Pipeline-Output/TCGA_WGS_Swag_Analysis/TCGA_Validation_2/TCGA-EJ-8474/1/somatic-sv-merging_v3.0/TCGA-EJ-8474_truvari_collapsed.vcf",
    "reference": null,
    "keep": "first",
    "bed": null,
    "gt": "off",
    "intra": false,
    "median_info": false,
    "debug": false,
    "refdist": 500,
    "pctseq": 0.0,
    "minhaplen": 50,
    "pctsize": 0.95,
    "pctovl": 0.0,
    "typeignore": false,
    "hap": false,
    "chain": false,
    "no_consolidate": true,
    "null_consolidate": null,
    "sizemin": 50,
    "sizemax": 50000,
    "passonly": false,
    "chunksize": 500,
    "bSample": null,
    "cSample": null,
    "sizefilt": 50,
    "no_ref": false
}
2024-10-01 11:40:09,280 [INFO] Zipped 346 variants Counter({'base': 346})
2024-10-01 11:40:09,280 [INFO] 66 chunks of 346 variants Counter({'__filtered': 312, 'base': 34})
2024-10-01 11:40:09,287 [INFO] Wrote 346 Variants
2024-10-01 11:40:09,287 [INFO] 0 variants collapsed into 0 variants
2024-10-01 11:40:09,287 [INFO] 0 samples' FORMAT fields consolidated
2024-10-01 11:40:09,287 [INFO] Finished collapse
```
I have tried multiple different parameters for `--keep` and `--pctseq` with no difference in the output. Any suggestions to resolve the same?

**Comment by ACEnglish** [comment]
According to the logs, 312 of the 346 variants are not within the size boundaries. Of the remaining 34, apparently none are similar enough to collapse. 

With the given information, this is all I can tell of your experiment. Are you sure there are variants that should collapse? Are your VCF entries formatted correctly and you expect 312 outside the size boundaries? [details on how size is determined](https://truvari.readthedocs.io/en/latest/truvari.html#truvari.entry_size)

**Comment by vondant** [comment]
What would be the best course of action recommended for merging somatic SV calls in this case?

**Comment by ACEnglish** [comment]
I'm unable to answer that question as I do not have the data.

**Comment by vondant** [comment]
My apologies, let me rephrase are there any specific parameters you would suggest turning off when merging somatic calls over germline ones?
